frontend/public/components/sidebars/explore-type-sidebar.tsx

```diff
@@ -51,7 +51,7 @@ export const ExploreType: React.FC<ExploreTypeProps> = (props) => {
   const description = currentSelection
     ? currentSelection.description
     : currentDefinition.description;
-  const required = new Set(currentDefinition.required || []);
+  const required = new Set(currentDefinition?.required || []);
```

Comment:
Could there be an error on line 53 as well or is `currentSelection` always set when `currentDefinition` is undefined?